app/src/main/res/values/strings.xml

```diff
@@ -26,154 +26,154 @@
     <string name="title_remember">Remember</string>
     <string name="title_export">Export</string>
     <string name="title_import">Import</string>
-    <string name="title_view_only_mode">View-only Mode</string>
-    <string name="title_hide_remote_cursor">Hide Remote Cursor</string>
-    <string name="title_key_compat_mode">Keyboard Compatibility Mode</string>
-    <string name="title_use_ssh_tunnel">Use SSH Tunnel</string>
-    <string name="title_unknown_ssh_host">Unknown SSH Host</string>
-    <string name="title_ssh_host_key_changed">Host Key changed!</string>
+    <string name="title_view_only_mode">View-only</string>
```

Comment:
```suggestion
    <string name="title_view_only_mode">View-only mode</string>
```
Should we keep `mode`? `View-only` looks too short to me, and we also have `Keyboard compatibility mode`.

Reply:
And "Compatiblity mode", "advanced mode", and "immersive mode".
While I don't see those doing anything here, "View-only mode" is the same in Remmina.

Reply:
I was just discussing. No need to point out all inconsistencies. 
As for Remmina, I can point towards bVNC, which uses "View-only mode".